admin-dev/themes/new-theme/js/components/pagination/dynamic-paginator.ts

```diff
@@ -290,15 +291,14 @@ export default class DynamicPaginator {
   }
 
   /**
-   * @param {Object} selectorsMap
+   * @private
```

Comment:
It's a bit redundant since we now have the `private` attribute, but nevermind

Reply:
All docblocks are redundant now. (we had this discussion in php when introduced types after 7.1 and we decided to still keep them) - if hoewever, you think we should remove them in ts, we should decide that before @NeOMakinG migrates everything :smile:

Reply:
Everything is already migrated except 20-30 files of 250+ so... 😆 

I feel like params are useful if you add description to it, what it does and what the goal of the param but that's it

Also, I think we'll have tons of conflicts on merging 1.7.8 into develop if we remove them all but well...